ingest-app/docs/api.md

```diff
@@ -694,9 +693,10 @@ curl -i -X DELETE \
 
 ### <a name="create-update-subscription"></a> Create / Update a Subscription
```

Comment:
I think we want to add a section separate for creating subscription without a native-id. It is confusing to have the POST creating without native-id mixed together with the UPDATE endpoint.